app/src/main/java/org/mifos/mobile/ui/beneficiary/presentation/BeneficiaryAddOptionsFragment.kt

```diff
@@ -27,6 +33,8 @@ import org.mifos.mobile.utils.Toaster
 @AndroidEntryPoint
 class BeneficiaryAddOptionsFragment : BaseFragment() {
 
+    private lateinit var composeView: ComposeView
```

Comment:
No need, remove it

Reply:
![Screenshot (128)](https://github.com/openMF/mifos-mobile/assets/90254566/08ee52fe-759d-411d-9e50-a7d7ddf5ef9e)